injective-events/convo.go

```diff
@@ -310,5 +310,11 @@ var cmd = codegen.Cmd
 var injectiveNetworkRegexp = regexp.MustCompile(`^injective`)
 
 func injectiveNetworks() []*registry.Network {
-	return networks.GetSubstreamsRegistry().Search(injectiveNetworkRegexp)
+	var result []*registry.Network
+	for _, network := range networks.GetSubstreamsRegistry() {
+		if injectiveNetworkRegexp.MatchString(network.ID) {
+			result = append(result, network)
+		}
+	}
+	return result
```

Comment:
@codegen-sh What is the reasoning of this change? Please explain into a reply to this exact comment.